QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150511_001997	$King Zachariel had left many arrangements for future generations. {nl}We and the Jonas family were one of them. {nl}But the meaning of it is only known by the Goddess. {nl}At least we know that the will of the king that we are protecting and the secret of the tomb indicates one person.
+QUEST_20150511_001998	$Trees become bigger as years pass. {nl}We can only see Woodman here but if you think of the large trees or the trees that stand in the old capital city… {nl}You can't help but think of the existence of even larger trees. {nl}After the disaster occurred, many plants turned into monsters so we can't ever rule out that possibility.
+QUEST_20150511_001999	$(There are barely legible words written here.) {nl}Euros could not hold his breath any more so he ended up inhaling the spores and died. {nl}Mushwort hunt people with the cruelest method I've ever seen. {nl}They walks around our gathering places and shakes out the mushrooms that would release spores.
+QUEST_20150511_002000	$We never know when it will release spores. If we don't stop it, the whole camp could be annihilated. {nl}I have no way to keep safe from it. I can't stop coughing…
+QUEST_20150511_002001	$The training officer completed the training at the camp in the woods of the attached bridge, in accordance with the order to prepare for future disasters. {nl}I hereby grant this certificate. 
+QUEST_20150511_002002	$The Mausoleum of Zachariel has never allowed anyone entry. {nl}If the object was not for the world to know of, then we could've hidden it so that none could find it, but instead we just built a giant tomb to protect it.{nl}Is this the great king's conceit or does it hold a different meaning?
+QUEST_20150511_002003	$For the first step: recharge the magical essence I gave to you. Look for where magic is condensed at Pavydo Mixed Forest. {nl}You won't be able to use the magical essence right away, however, since it has also absorbed some malign energy. {nl}For that, you'll need to collect Operor saliva and purify it at the altar of purification at the Wasteland of Renistu.
+QUEST_20150511_002004	$Name: High Raven{nl}Sex: Male{nl}Occupation: Merchant{nl}List of items: luxurious goods, weapons {nl}{nl}In accordance with kingdom laws, you may enter.
+QUEST_20150511_002005	$If you're seeing this you either didn't pay me in full or did something I told you not to do. {nl}I have cursed this letter. {nl}I've no idea what's gonna happen to you from now on, but know that my curse is absolute.
+QUEST_20150511_002006	$The one who fell while investigating the knowledge. {nl}Return to the Goddess by becoming a part of the canyon. {nl}The saddened mind breaks the gold stones of the canyon and the poured alcohol soaks into the ground on behalf of the tears.
```

Comment:
So that's one of the poems, eh.
I'm not quite sure, but isn't if "on behalf of some**one**"? I think you should use "instead of tears" here.